sourcecoop/egg-source-coop.json

```diff
@@ -0,0 +1,152 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-05-14T22:54:17-07:00",
+    "name": "SourceCoop",
+    "author": "red_thirten@yahoo.com",
+    "description": "SourceCoop is a cooperative server-side-only mod for multi-player Source Engine games (like Black Mesa), that enables players to play together on single-player campaigns or maps. This Egg currently supports official Black Mesa maps only, but can be easily modified to support more (see Egg README for details).",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/steamcmd:debian": "ghcr.io\/parkervcp\/steamcmd:debian"
```

Comment:
This is a source game so should it not use the source image? (not that their diffrend at all :) )

Reply:
no its fine, leave it at this as it is barely used.